lib/models/blueprint.js

```diff
@@ -1630,11 +1701,31 @@ function finishProcessingForInstall(infos) {
   return sequence(infosNeedingConfirmation).then(() => infos);
 }
 
-function finishProcessingForUninstall(infos) {
-  let validInfos = infos.filter((info) => fs.existsSync(info.outputPath));
+function finishProcessingForUninstall(infos, shouldTransformTypeScript) {
+  let validInfos = infos.filter((info) => {
+    if (shouldTransformTypeScript && isTypeScriptFile(info.outputPath)) {
```

Comment:
Consider (and let me know) if we should incorporate the `--typescript` flag value here also.